packages/flame_tiled/lib/src/renderable_layers/renderable_layer.dart

```diff
@@ -109,8 +109,16 @@ abstract class RenderableLayer<T extends Layer> {
     final anchor = camera.viewfinder.anchor;
     final cameraX = camera.viewfinder.position.x;
     final cameraY = camera.viewfinder.position.y;
-    final viewportCenterX = camera.viewport.size.x * anchor.x;
-    final viewportCenterY = camera.viewport.size.y * anchor.y;
+    final viewportCenterX = camera.viewport.size.x / 2;
+    final viewportCenterY = camera.viewport.size.y / 2;
+
+    if (parent is GroupLayer) {
```

Comment:
@spydon check this out. I am pretty sure this isn't right, but I got the test working by doing two changes.

The column was getting out of place because it have a parallax factor
<img width="273" alt="Screenshot 2023-05-25 at 22 27 46" src="https://github.com/flame-engine/flame/assets/835641/dd5a4adb-2fc6-4b2f-a2d3-3086849005a9">

And the viewport size was zero, so the parallax was being calculated wrongly.

I believe that the viewport size should never be zero right? Even if it isn't a fixed one, it should resort to the canvas, for the test to pass, I hard coded the viewport size on the test here: https://github.com/flame-engine/flame/pull/2549/files#diff-92d1b7e7c2fa039cbccdba852c50d50aab7c0db8f1a206c03d8597ab097cb3c9R415

But we probably want that the viewport fallback to the canvas size when a fixed one isn't defined?

I also reverted this [change](https://github.com/flame-engine/flame/pull/2538/files#diff-461c0b715937055337c5a06eee3cfd5e3b3768525e098d217c8964eda5c8a0aaR112), because a top left anchor is x = 0, y =0. If we want the center of the viewport, shouldn't we always divide by two? Or maybe I am missing something, we need to keep the anchor into consideration but we should change the anchor to center on the test?

Idk, it is late and my head can't think straight anymore. But hopefully when you read this, some of it will make sense, or maybe the Erick of tomorrow, well rested will also be able to make sense of all this.

Reply:
I think that we have to take the viewfinder anchor into consideration? But otherwise it looks correct.

Reply:
@spydon I am unsure about that, I feel that the camera anchor sounds affect the tiled parallax. The camera should only affect how the map is seen in the game, not how it is rendered. If my argument makes sense to yo, feel free to merge this tomorrow, otherwise, I am no tiled expert, so I am not really sure how to take it in account, so maybe we should seek for help here?

Reply:
> The camera should only affect how the map is seen in the game, not how it is rendered

@erickzanardo I'm not sure I follow, the camera is passed in here so that it can affect how the map is seen by the camera, the parallax effect by definition can't be statically rendered up front since it's causes the layers to change relative to one another based on the camera's current position and zoom

Reply:
Humm, yeah, I think that makes sense, I don't have an advanced knowledge on Tiled so I still get confused about those "more advanced" features from it.

But in fact a parallax needs to be affected by the camera, otherwise the parallax effect will not happen!

Either way, if we dismiss that statement of mine, the camera position and zoom is been taken into account, the only missing key here is about how we could apply the anchor too, and if it should be applied.

Reply:
move the discussion to another comment https://github.com/flame-engine/flame/pull/2549#discussion_r1219505747